docs-src/references/web-ui-configuration.md

```diff
@@ -204,3 +158,89 @@ Prevents the user from resetting Workflows from the Web UI.
 ```yaml
 workflowResetDisabled: false
 ```
+
+## batchActionsDisabled
+
+Prevents the execution of Batch actions.
+
+```yaml
+batchActionsDisabled: false
+```
+
+## hideWorkflowQueryErrors
+
+Hides any errors resulting from a Query to the Workflow.
+
+```yaml
+hideWorkflowQueryErrors: false
+```
+
+## cors
+
+The name of the `cors` field stands for Cross-Origin Resource Sharing.
+Use this field to provide a list of domains that are authorized to access the UI Server APIs.
+
+```yaml
+cors:
+  cookieInsecure: false
+  allowOrigins:
+    - http://localhost:3000 # used at development by https://github.com/temporalio/ui
+```
+
+## tls
+
+Transport Layer Security (TLS) configuration.
+
+```yaml
```

Comment:
should this example have some default example values like the auth configuration does?

Reply:
The defaults are all empty strings.

Reply:
Finally got a reply. Fixing this one right up.